src/annotator/config/config-func-settings-from.js

```diff
@@ -1,3 +1,9 @@
+/**
+ * @typedef HypothesisWindowProps
+ * @prop {() => Object} [hypothesisConfig] - Function that returns configuration
+ *   for the Hypothesis client
```

Comment:
`Object` doesn't provide useful details on _what_ config properties can be returned here. Filling that out is an exercise for the future.

Reply:
Should we leave a note?